core/src/iosSimulatorArm64Test/kotlin/com/powersync/testutils/TestUtils.iosSimulatorArm64.kt

```diff
@@ -13,3 +13,5 @@ actual fun cleanup(path: String) {
         SystemFileSystem.delete(resolved)
     }
 }
+
+actual fun getTempDir(): String? = null
```

Comment:
Maybe add a comment explaining that while we could use `SystemTemporaryDirectory` here, we're deliberately not doing that because SQLiter doesn't support setting a directory and that tests would attempt to do so if `getTempDir()` were to return something.

Reply:
Going point! I've added a comment to explain this.